server/server.go

```diff
@@ -376,7 +376,8 @@ func (s *Server) KillAllConnections() {
 	log.Info("[server] kill all connections.")
 
 	for _, conn := range s.clients {
-		killConn(conn, false)
```

Comment:
You can still cancel the executing, the client conn is closed, so the wrong result will not send to the client.

Reply:
I think no need to do this.

Reply:
I agree with winkyao.